src/app/clusters/camera-av-stream-management-server/camera-av-stream-management-server.cpp

```diff
@@ -146,18 +146,6 @@ CHIP_ERROR CameraAVStreamMgmtServer::Init()
                          mEndpointId));
     }
 
```

Comment:
Why was this check removed?

Reply:
There's a check for if any of the image control attributes, then is the feature set.  Which follows the pattern for other feature flag dependent optional attributes.

Reply:
Sorry, saw that it was added back again. Thanks. Resolving.